build.sh

```diff
@@ -1,14 +1,27 @@
 #!/bin/bash
 echo "Build Script for eduVPN (and derivatives)"
+
+#Check if HomeBrew Installed
```

Comment:
as far as I know, installing homebrew requires root permissions. Maybe the script could just stop and ask the user to manually install homebrew before running the script again. The build script can be run as a normal user, right?

Reply:
HI François Boss,
Ok, I will ask user to install homebrew manually 
.
Yes the build script can be run as normal user